tests/ga/test_update.py

```diff
@@ -156,13 +156,13 @@ def check_running(*val, **__):
                                 type(update_handler).is_running = True
 
 
-class UpdateTestCase(AgentTestCase):
+class UpdateTestCase(AgentTestCaseWithGetVmSizeMock):
     _test_suite_tmp_dir = None
     _agent_zip_dir = None
 
     @classmethod
     def setUpClass(cls):
-        AgentTestCase.setUpClass()
+        super(UpdateTestCase, cls).setUpClass()
```

Comment:
There are a couple of these scattered throughout the code base, but I've only updated the ones that would be broken by the new intermediate base class.